packages/editor/src/components/editor/index.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { Notice } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { TEMPLATE_POST_TYPE } from '../../store/constants';
+import EditorInterface from '../editor-interface';
+import EditorProvider from '../provider';
+import Sidebar from '../sidebar';
+
+function Editor( {
+	postType,
+	postId,
+	templateId,
```

Comment:
The templateId can in theory be guessed from the postType, postId couple but it would require a lot more changes to how things work now in the site editor. So for now, this can work too.